tripy/tripy/frontend/module/groupnorm.py

```diff
@@ -70,15 +70,17 @@ def __init__(self, num_groups: int, num_channels: int, dtype: datatype.dtype = d
             group_norm.weight = tp.ones_like(group_norm.weight)
             group_norm.bias = tp.zeros_like(group_norm.bias)
 
-            input = tp.iota((1, 4, 2, 2))
+            input = tp.iota((1, 4, 2, 2), dim=1)
```

Comment:
We should be adding unit tests for this case rather than changing the docstring example.

Reply:
The reason for this change is because `tp.iota((1, 4, 2, 2))` is simply equivalent to `tp.zeros((1, 4, 2, 2)),` which is not very useful for either demoing how GroupNorm works and docstring tests.